src/Aquifer.Public.API/Endpoints/Bibles/Texts/Get/Request.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Aquifer.Public.API.Endpoints.Bibles.Texts.Get;
+
+public class Request
+{
+    /// <summary>
+    /// The ID of the Bible for which to return text.
+    /// </summary>
+    public int BibleId { get; init; }
+
+    /// <summary>
+    /// Book code based off USFM book identifier (e.g. GEN, EXO, etc.). Can get a list of available books and identifiers
+    /// from the /bible-books endpoint. This parameter is required.
+    /// </summary>
+    public string? BookCode { get; init; }
```

Comment:
If it's required from the validator then no reason for it to be null. Then that removes the need for the bang operator in the endpoint.

Reply:
Good call: [fix commit](https://github.com/BiblioNexusStudio/aquifer-server/pull/443/commits/a6799102add9ff609464cdead5ab9395c522df78).